src/contentsharecontroller/ContentShareControllerFacade.ts

```diff
@@ -12,6 +13,12 @@ export default interface ContentShareControllerFacade {
    */
   setContentAudioProfile(audioProfile: AudioProfile): void;
 
+  enableSimulcastForContentShare(
+    enable: boolean,
+    lowLayerEncodingParams?: VideoEncodingParameters,
```

Comment:
there might be a chance we add a midLayer in the future. I did think about adding its as an array but then the order would be 0 - 'low' and 1 - 'high' (similar to the order in `SimulcastTransceiverController`) so when we extend the `mid` layer it would be like index 2 which is kinda weird.
How about I create a new type like below instead?
```
type ContentShareSimulcastEncodingParameters = {
  lowLayer: VideoEncodingParameters;
  highLayer: VideoEncodingParameters;
}
```

Reply:
Adding a type or an interface will work. Parameters will also work if we make `ContentShareControllerFacade.enableSimulcastForContentShare` an internal method. We can freely change the order of parameters in the future.